app/views/ChargeSummary.scala.html

```diff
@@ -48,10 +50,10 @@
 )
 @(currentDate: LocalDate,
     documentDetailWithDueDate: DocumentDetailWithDueDate, backUrl: String,
-    paymentBreakdown: List[FinancialDetail], chargeHistory: List[ChargeHistoryModel], paymentAllocations: List[PaymentHistoryAllocations],
+    paymentBreakdown: List[FinancialDetail], paymentAllocations: List[PaymentHistoryAllocations],
```

Comment:
next story created to refactor this view:
[MISUV-7901](https://jira.tools.tax.service.gov.uk/browse/MISUV-7901)